src/dotnet/Notification.Service.Migration/NotificationDbContextContextFactory.cs

```diff
@@ -0,0 +1,21 @@
+using ActualChat.Notification.Db;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Design;
+
+namespace ActualChat.Notification.Migrations;
+
+public class NotificationDbContextContextFactory : IDesignTimeDbContextFactory<NotificationDbContext>
+{
+    public string UsePostgreSql =
+            "Server=localhost;Database=ac_dev_notification;Port=5432;User Id=postgres;Password=postgres";
```

Comment:
I noticed that we our connection strings are hardcoded at many places. Just trying to catch the idea...

Reply:
default values for integration tests. But I agree, I also don't like it. Let's discuss